includes/class-boldgrid-inspirations-deploy.php

```diff
@@ -1427,6 +1439,9 @@ public function deploy_page_sets() {
 
 					if ( $term_id ) {
 						wp_set_object_terms( $post_id, $term_id, $tax_data->taxonomy );
+						$theme_has_cph = true;
```

Comment:
I would suggest we update the logic. e.g. Just because we have a $term_id doesn't mean the theme has custom page headers. Maybe at this moment, yes.